apps/ui/pages/reservation-unit/[id].tsx

```diff
@@ -518,26 +623,36 @@ const ReservationUnit = ({
 
   const [shouldCalendarControlsBeVisible, setShouldCalendarControlsBeVisible] =
     useState(false);
-
+  const isClientATouchDevice = isTouchDevice();
   const handleCalendarEventChange = useCallback(
     (
       { start, end }: CalendarEvent<ReservationType>,
-      skipLengthCheck = false
     ): boolean => {
       if (!reservationUnit) {
         return false;
       }
-      const newReservation = getNewReservation({ start, end, reservationUnit });
-
       if (
-        !isSlotReservable(start, end, skipLengthCheck) ||
+        !isSlotReservable(
+          start,
+          end,
+          reservationUnit,
+          activeApplicationRounds
+        ) ||
```

Comment:
This looks wrong. The same function is called with `skipLengthCheck` at the third position. ex. line 700

Reply:
I ended up removing the `isSlotReservable()` function all together. It returns in practice the same thing as `isReservationReservable()`, so using the latter instead.